development/tools/cmd/githubissues/README.md

```diff
@@ -0,0 +1,33 @@
+# Github issues
+
+
+## Overview
+This command queries all open Github issues in an organization or repository, and loads that data to a BigQuery table.
+### Flags
+
+See the list of available flags:
+
+| Name                             | Required | Description                                                                                          |
+| :-----------------------------   | :------: | :--------------------------------------------------------------------------------------------------- |
+| **--githubOrgName**              |   Yes    | The string value with the Github organization name to retrieve issues from.
+| **--githubRepoName**             |    No    | The string value with the Github repository name to retrieve issues from.
+| **--githubToken**                |   Yes    | The string value with the Github OAuth token.
+| **--githubBaseURL**              |    No    | The string value with the custom Github API base URL.
+| **--issuesFilename**             |    No    | The string value with the name of the generated file with list of issues. It defaults to `issues.json`.
+| **--bqCredentials**              |   Yes    | The string value with the path to BigQuery credentials JSON file.
+| **--bqProjectID**                |   Yes    | The string value with the name of the BigQuery project.
+| **--bqDatasetName**              |   Yes    | The string value with the name of the BigQuery dataset.
+| **--bqTableName**                |   Yes    | The string value with the name of the BigQuery table.
+
+## Creating empty table
+This program assumes the table already exists. In order to create new table:
+
+* Go to BigQuery console
+* Create new table in a dataset
+* Edit schema as text
+  * For organization copy schema from `table_org_schema.json`
+  * For singular repo copy schema from `table_repo_schema.json`
+* Select partitioning on `updated_at` field
+
+## Error during upload
+Bigquery will fail if the JSON file is bigger than 100MB. The file can be split into smaller parts and uploaded manually before rerunning the program to fix that issue.
```

Comment:
```suggestion
If the JSON file is bigger than 100MB, Bigquery fails. To fix that issue, you can split the file into smaller parts and upload them manually before rerunning the program.
```

Shouldn't this information come *before* users try to run the program? Is there any advantage to let users first experience the error and make them run it once more afterwards?

Reply:
Files can be bigger than 100MiB for organizations with many issues (`kyma-project` JSON file had around 200MB, since it contains info about over 21 thousand issues and PRs that were created), when the program was never run before. Subsequent runs will generate way smaller JSON files containing only differences since last update.
Even if the program fails, it will generate JSON file that can be split into smaller parts and added manually.